src/CMakeLists.txt

```
@@ -0,0 +1,28 @@
+if (NOT TARGET acquire-core-logger)
+    add_subdirectory(acquire-core-libs)
+endif ()
+
+set(tgt acquire-driver-spinnaker)
+
+if (TARGET spinnaker)
+    add_library(${tgt} MODULE spinnaker.cpp)
+    target_link_libraries(${tgt} PRIVATE
+            acquire-core-logger
+            acquire-core-platform
+            acquire-device-kit
+            spinnaker
+            )
+else ()
+    add_library(${tgt} MODULE unsupported.driver.c)
+    target_link_libraries(${tgt} PRIVATE
+            acquire-core-logger
+            acquire-core-platform
+            acquire-device-kit
+            )
+endif ()
+
+set_target_properties(${tgt} PROPERTIES
+        MSVC_RUNTIME_LIBRARY "MultiThreaded$<$<CONFIG:Debug>:Debug>"
+        )
```

See my comment in `git_versioning.cmake`

```suggestion
target_add_git_versioning(${tgt})
set_target_properties(${tgt} PROPERTIES
        MSVC_RUNTIME_LIBRARY "MultiThreaded$<$<CONFIG:Debug>:Debug>"
        )
```